src/content_script/index.tsx

```diff
@@ -194,8 +200,8 @@ async function main() {
             e.preventDefault()
             let text = (window.getSelection()?.toString() ?? '').trim()
             if (!text) {
-                if (event.target instanceof HTMLInputElement || event.target instanceof HTMLTextAreaElement) {
-                    const elem = event.target
+                if (e.target instanceof HTMLInputElement || e.target instanceof HTMLTextAreaElement) {
```

Comment:
I am not sure if `e.target instanceof` will work or not in the plugin context since HTMLInputElement should be instances of the current Window context.